DependencyInjection/Configuration.php

```diff
@@ -105,6 +105,7 @@ public function getConfigTreeBuilder()
                         ->scalarNode('default_format')->defaultNull()->end()
                         ->scalarNode('prefix_methods')->defaultTrue()->end()
                         ->scalarNode('include_format')->defaultTrue()->end()
+                        ->scalarNode('routing_ignore_classes')->defaultValue([])->end()
```

Comment:
What about `ignored_types`?

Reply:
Sorry I meant renaming the option to `ignored_types` because it also accepts interfaces and traits FQCN.